ingestion/src/metadata/orm_profiler/orm/converter.py

```diff
@@ -100,6 +100,7 @@ def build_orm_col(idx: int, col: Column, table_service_type) -> sqlalchemy.Colum
         type_=_TYPE_MAP.get(col.dataType),
         primary_key=not bool(idx),  # The first col seen is used as PK
         quote=check_snowflake_case_sensitive(table_service_type, col.name.__root__),
+        key=str(col.name.__root__).lower(),
```

Comment:
what does this do?

Reply:
It is for snowflake. Basically when we create the declarative table we'll set column name uppercase, though when running the `from_statement` it will use the keys from the textualAnnotation object (i.e. the object created from the `text(<raw_sql>)` which will be lowercase. This way by passing forcing lowercase key on the declarative table we get a match. Otherwise we get <column>  not found.

Reply:
oh wow interesting! Could you please add a line in the code itself explaining that?